### PR TITLE
fix(abg): do not use double slash for top-level action URL

### PR DESCRIPTION
**Pull request by Vampire**

### CODE REVIEW

**Comment by Vampire** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/typesafegithub/github-workflows-kt/1599?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1600** <a href="https://app.graphite.dev/github/pr/typesafegithub/github-workflows-kt/1600?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1599** <a href="https://app.graphite.dev/github/pr/typesafegithub/github-workflows-kt/1599?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1598** <a href="https://app.graphite.dev/github/pr/typesafegithub/github-workflows-kt/1598?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Vampire and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: action-binding-generator/src/test/kotlin/io/github/typesafegithub/workflows/actionbindinggenerator/typing/TypesProvidingTest.kt [hunk 1]
```diff
@@ -103,7 +103,7 @@ class TypesProvidingTest :
                 // Given
                 val fetchUri: (URI) -> String = {
                     when (it) {
-                        URI("https://raw.githubusercontent.com/some-owner/some-name/some-hash//action-types.yml") -> hostedByActionYml
+                        URI("https://raw.githubusercontent.com/some-owner/some-name/some-hash/action-types.yml") -> hostedByActionYml
```
Comment (krzema12) [inline action-binding-generator/src/test/kotlin/io/github/typesafegithub/workflows/actionbindinggenerator/typing/TypesProvidingTest.kt:106]: I'm afraid we don't have an unit test here that covers subactions. Could you add one please, to ensure we don't break things?

Comment (Vampire) [reply action-binding-generator/src/test/kotlin/io/github/typesafegithub/workflows/actionbindinggenerator/typing/TypesProvidingTest.kt:106]: One? One per case. :-) Added

**Review by krzema12 — APPROVED** [review]

**Comment by krzema12** [comment]
### Merge activity

* **Aug 31, 1:09 PM EDT**: @krzema12 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/typesafegithub/github-workflows-kt/1599).

**Pull request merged**